### ISSUE TITLE
support for auto bhop

### ISSUE THREAD

**Issue by Veal-Uneasily4** [Theggv/Kreedz#50]
thanks for all of the work!
I'm using this auto bhop plugin: [](https://dev-cs.ru/resources/1221/)
It's possible to make that when a player **used auto bhop**, it will not put his record on /top?
or even better, count on him on top, but add [auto-bhop] before his name?
e.g. [auto-bhop] Player52 

thanks!

**Comment by Psycrow101** [comment]
@Veal-Uneasily4 How about pausing the player's progress when bhop is turned on, as is done for noclip?

**Comment by Veal-Uneasily4** [comment]
> @Veal-Uneasily4 How about pausing the player's progress when bhop is turned on, as is done for noclip?

Not really, cause then I guess, there's not really a point using the autobhop without count
